packages/bitcore-wallet-service/test/integration/server.js

```diff
@@ -9805,6 +9805,347 @@ describe('Wallet service', function() {
     });
   });
 
+  describe('Changelly', () => {
```

Comment:
move to a new file: test/integration/changelly.js

Reply:
Done